helpick-service/src/test/java/com/infoshare/service/VolunteerServiceTest.java

```diff
@@ -0,0 +1,29 @@
+package com.infoshare.service;
+
+import com.infoshare.database.DB;
+import com.infoshare.database.FileDb;
+import com.infoshare.domain.TypeOfHelp;
+
+import static org.junit.jupiter.api.Assertions.*;
+import org.junit.jupiter.api.Test;
+
+
+class VolunteerServiceTest {
+
+    private final DB db = new FileDb();
+    private final VolunteerService volunteerService = new VolunteerService(db);
+
+    @Test
+    public void ifNewVolunteerRegisteredForValidInput() {
+        assertTrue(volunteerService.registerNewVolunteer("Janko", "Gdansk", "janko@gmail.com", "123654789", TypeOfHelp.SHOPPING, true));
+        db.deleteVolunteer("janko@gmail.com");
```

Comment:
Ta operacja nie powinna się znaleźć w teście.
Gdyby jednak bardzo zależało, na sprawdzeniu czy w bazie pojawił się zarejestrowany wolontariusz, to lepiej użyć np. dodatkowej asercji:
`assertThat(db.getVolunteer("janko@gmail.com")).isNotNull();`